### PR TITLE
[Improve] [Flink-Kubernetes-V2] Change doTriggerSavepoint log

### PR DESCRIPTION
**Pull request by caicancai**
## What changes were proposed in this pull request

Change doTriggerSavepoint log

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)
 - no

### CODE REVIEW

**Comment by caicancai** [comment]
cc @wolfboys @Al-assad

File: streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/trait/KubernetesClientV2Trait.scala [hunk 1]
```diff
@@ -160,12 +160,12 @@ trait KubernetesClientV2Trait extends FlinkClientTrait {
       }
       .runIOAsTry match {
       case Success(rsp) =>
-        logInfo(richMsg("Cancel flink job successfully."))
+        logInfo(richMsg("TriggerSavepoint flink job successfully."))
```
Comment (Al-assad) [inline streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/trait/KubernetesClientV2Trait.scala]: change to "Trigger savepoint of flink job xx"

**Review by Al-assad — CHANGES_REQUESTED** [review]

File: streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/trait/KubernetesClientV2Trait.scala (hunk 1 above)
Comment (caicancai) [reply streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/trait/KubernetesClientV2Trait.scala]: done

**Review by wolfboys — APPROVED** [review]
LGTM

**Review by Al-assad — APPROVED** [review]
LGTM

**Pull request merged**